x/oracle/README.md

```diff
@@ -78,8 +78,8 @@ The reward pool is not distributed all at once, but instead over a period of tim
 A `VotePeriod` during which either of the following events occur is considered a "miss":
 
 - The validator fails to submits a vote for **each and every** exchange rate specified in `AcceptList`.
-
 - The validator fails to vote within the `reward band` around the weighted median for one or more denominations.
+- Validators don't provide enough price votes for a given denom (ballot denom support is less than the required `VoteThreshold`) - in that case all validators will receive a miss.
```

Comment:
Can you make this point in **BOLD Text**

```suggestion
- Validators don't provide enough price votes for a given denom (ballot denom support is less than the required `VoteThreshold`) - **in that case all validators will receive a miss.**
```

Reply:
OK, good idea